src/ingest-pipeline/airflow/dags/utils.py

```diff
@@ -1050,13 +1053,45 @@ def _get_workflow_map() -> List[Tuple[Pattern, Pattern, str]]:
     return COMPILED_WORKFLOW_MAP
 
 
+def _get_type_client() -> TypeClient:
+    """
+    Lazy initialization of the global TypeClient instance
+    """
+    global TYPE_CLIENT
+    if TYPE_CLIENT is None:
+        conn = HttpHook.get_connection('search_api_connection')
+        if conn.port is None:
```

Comment:
Is it `conn.conn_type` or `conn.type`? There's a mismatch between the `if`/`else` branches.

To eliminate that possibility, maybe something like:
```python
port_str = '' if conn.port is None else f':{conn.port}'
url = f'{conn.type}://{conn.host}{port_str}'
```

(with either `conn.type` or `conn.conn_type` as appropriate)

Reply:
Ooh, good catch, thanks!